contrib/initramfs/Makefile.am

```diff
@@ -6,16 +6,14 @@ initrd_SCRIPTS = \
 SUBDIRS = hooks scripts
 
 EXTRA_DIST = \
-	$(top_srcdir)/etc/init.d/zfs \
-	$(top_srcdir)/etc/init.d/zfs-functions \
+	$(top_srcdir)/etc/init.d/zfs.in \
```

Comment:
Why are you adding the `.in` files to EXTRA_DIST here? They are already included in the tarball via the entries in `etc/init.d/Makefile.am`. If you just outright remove them here, does it still work?

Reply:
Yea, I think you're right that the .in files should not be here.